packages/flutter_tools/gradle/aar_init_script.gradle

```diff
@@ -8,7 +8,24 @@ import org.gradle.api.Project
 import org.gradle.api.artifacts.Configuration
 import org.gradle.api.publish.maven.MavenPublication
 
-void configureProject(Project project, String outputDir) {
+initscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.yaml:snakeyaml:2.2+'
```

Comment:
We probably do not want to add a new gradle library dependency.

Reply:
It's easy to parse YAML using this library, but parsing YAML manually is difficult. If library usage isn't feasible, I'll attempt manual parsing.

Reply:
Hold off on that for now. Let us review the idea of the pr and then we can take a look at yaml parsing. I dont want you to write manual yaml parsing if this cl has other things that prevent us from merging.